services/backend/src/core/search/util/getAllProfiles.js

```diff
@@ -398,6 +398,10 @@ async function getAllUsers(language, userId, request) {
         : undefined;
     }
 
+    if (info.exFeeder) {
+      info.exFeederText = "ExFeeder Ex-Feeder";
+    }
+
```

Comment:
we also need the french. according to mena it has another term. please confirm with her